src/mango/src/mango_execution_stats.erl

```diff
@@ -35,6 +37,15 @@ to_json(Stats) ->
         {execution_time_ms, Stats#execution_stats.executionTimeMs}
     ]}.
 
+to_map(Stats) ->
```

Comment:
The names of the fields are quite long. On the other hand they are crystal clear and it would be easy to read the logs and analyze the logs.

I don't have any opinion. Therefore, I would just state the fact. 

The current choice of names takes up at least 248 bytes in the log message (including date).

Reply:
Yeah... I thought about making shorter versions of the names, but those are the names as they are in the codebase, and gzip'ed log rotation reduces the impact of field length. I think more importantly though, we've already introduced those names into the public API as documented in the `Execution Statistics` portion of the `_find` documentation [1] and so I think it makes sense to stay consistent with those, or at the very least, we should update those as well if we're going to make shorter names here.


[1] https://docs.couchdb.org/en/stable/api/database/find.html#find-statistics